lab-michael/lib/storage.js

```diff
@@ -37,19 +39,20 @@ exports.fetchItem = function(recipe, id) {
     } catch (err) {
       return Promise.reject(err);
     }
-  })
-  .catch(err => Promise.reject(err));
+  });
 };
 
 exports.deleteItem = function(recipe, id) {
   return new Promise((resolve,reject) => {
     if (!recipe) return reject (new Error('expected recipe'));
     if (!id) return reject (new Error('expected id'));
+    // console.log(recipe.id);
+    // console.log(recipe);
+    resolve(recipe);
```

Comment:
What's the value of resolving this recipe here when you return an fs.unlink (Promise object) below? Take another look at this and see if you can find anything way to manage the flow of data.